scraper.rb

```diff
@@ -1,14 +1,22 @@
 #!/bin/env ruby
 # encoding: utf-8
 require 'scraperwiki'
-require 'colorize'
-require 'combine_popolo_memberships'
+require 'require_all'
+require 'scraped_page_archive'
 
-require_relative 'lib/members.rb'
+require_rel 'lib'
 
 members_url = 'http://api.openhluttaw.org/en/memberships'
-members = Members.new(members_url).to_h[:members_of_the_lower_house]
 
-members.each do |member|
-  ScraperWiki.save_sqlite([:id], member.to_h)
+def all_members(url)
```

Comment:
This is oddly named. It seems like it should be a query method, but this is actually a command method, with no return value, whose job is to save the data.